package.json

```diff
@@ -75,11 +75,13 @@
     "@babel/preset-typescript": "^7.8.3",
     "@babel/register": "^7.8.3",
     "@carbon/icons-react": "^10.8.1",
+    "@types/http-errors": "^1.6.3",
```

Comment:
I think that both @types/http-errors and @types/react (my bad) should go devdeps. What do you think?